composer.json

```diff
@@ -4,13 +4,10 @@
     "license": "MIT",
     "config": {
         "preferred-install": "dist",
-        "sort-packages": true,
-        "platform": {
-            "php": "7.0.25"
```

Comment:
You should keep the platform section. Just raise it to a 7.1 version, e.g. 7.1.3 (which is the minimum for Symfony 4 and Laravel 5.6).

Reply:
Thank you, @derrabus, updating!